examples/music-player/src/3_HomePage.tsx

```diff
@@ -62,7 +62,7 @@ export function HomePage({ mediaPlayer }: { mediaPlayer: MediaPlayer }) {
     ) => {
         if (!playlist) return;
 
-        playlist.title = evt.target.value;
+        updatePlaylistTitle(playlist, evt.target.value);
```

Comment:
The React Compiler doesn't like the Jazz way of updating data:

<img width="931" alt="Screenshot 2024-10-14 at 21 47 29" src="https://github.com/user-attachments/assets/094bd910-2e9c-4a89-8614-7dfd81c9cd63">

Fixed by moving all the updates into `actions`